app/api/v1/endpoints/tag.py

```
@@ -0,0 +1,18 @@
+from dependency_injector.wiring import Provide, inject
```

Ruff (0.8.2)

`dependency_injector.wiring.Provide` imported but unused

Remove unused import: `dependency_injector.wiring.Provide`

(F401)